.github/workflows/build.yml

```
@@ -12,24 +12,29 @@ on:
 env:
   MIX_ENV: ci
 
+# The approach to the CI matrix is as following:
+# * The minimum version of elixir is the minimum supported (see https://hexdocs.pm/elixir/compatibility-and-deprecations.html)
+# * Each Elixir version is paired with the lowest supported OTP version
+# * The latest Elixir version is paired also with the highest supported OTP version
+
 jobs:
   ci:
     strategy:
       fail-fast: false
       matrix:
-        include:
-          - elixir: '1.7'
-            otp: '20'
-          - elixir: '1.8'
-            otp: '20'
-          - elixir: '1.9'
-            otp: '20'
+        include:         
```

Looks like there's some trailing whitespace on this line, oops. 😅 

```suggestion
        include:
```